Engine/Make/_main_.env

```diff
@@ -480,3 +480,65 @@ cat >> $_File << EOF
 EOF
 }
 
+_t4dMakeGeneratorVscodeBuiltins(){
+# Generated From $Tools4Dev_PATH/Engine/G4d/templates-project.env
+# v1.8.0
+###### _t4dMakeGeneratorVscodeBuiltins
+#   - Name:
+#       _t4dMakeGeneratorVscodeBuiltins
+###
+#   - Synopsis:
+#       Generate builtins when BuiltinsVSCode equals true in main file and add output file in Utils/T4D-Plugin-VSCode
+###
+#   - Note:
+#       <Required>
+#       [Optionnal]
+#       {XXXXXXX}       = Value To Be Changed, if {URL} then replace {URL} with a valid URL
+###
+###### DOCUMENTATION BLOC #####
+########### CODE BLOC #########
+    local _t4dErrorCode=1
+
+    local _mainFiles="$(cat $Tools4Dev_PATH/.Ressources/index/private.index | cut -d ':' -f1 | rev | cut -d "/" -f 2- | rev | uniq | grep -v "^#")"
+    local _truefile=""
+    local _FileJson="$Tools4Dev_PATH/Utils/T4D-Plugin-VSCode/snippets/builtinst4d.json"
```

Comment:
please structure your folder with a better architecture than "EverythingInThatBag" or el famoso "AllinOneFile" :) 

you may want to cut by file or Type Like Engine/Source/ etc.